src/commands/fix-mismatches.ts

```diff
@@ -17,7 +17,8 @@ const getWorkspaceVersion = (name: string, wrappers: SourceWrapper[]): string |
 };
 
 export const fixMismatches = (wrappers: SourceWrapper[], options: Options): void => {
-  const iterator = getMismatchedDependencies(wrappers, options);
+  // TODO: decide what to do here
+  const iterator = getMismatchedDependencies(wrappers, { ...options, matchRanges: false });
```

Comment:
This one requires the new option, so I made it work exactly as before - is that good enough?